<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGameDialog</class>
 <widget class="QDialog" name="CreateGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lblMaxPlayers">
     <property name="text">
      <string>Maximum players</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="leRoomName"/>
   </item>
   <item row="2" column="1">
    <spacer name="hsMaxPlayers">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblRoomName">
     <property name="text">
      <string>Room name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="sbMaxPlayers">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="bbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="cmbSystem"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSystem">
     <property name="text">
      <string>Game system</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
